app/components/archive-bucket-dialog/archive-bucket-dialog.html

```diff
@@ -22,7 +22,7 @@
       </p>
 
       <div class="archive-bucket-dialog__final-caution">
-        Caution: This cannot be undone
+        Info: This bucket can be unarchived later
       </div>
```

Comment:
this should be a conditional depending on if the bucket was fully funded! because we can't un-archive a bucket whose funds have been returned to the users...